bench/CMakeLists.txt

```diff
@@ -6,6 +6,14 @@ function(add_benchmark BM_NAME)
     # target_link_options(${BM_NAME} PRIVATE -Wall -march=native -fsanitize=address)
     target_link_libraries(${BM_NAME} PRIVATE graph-prototype refl-cpp fmt ut)
     set_project_warnings(${BM_NAME})
+
+    add_executable(${BM_NAME}_nosimd ${BM_NAME}.cpp)
```

Comment:
... would be nicer to have some (not all) of the benchmark 1 examples being w/ and w/o simd. We plan to have many more benchmark tests and this duplication isn't always necessary desired ... i.e. IMO just specific for the bm_case1 and not for bm_case0 or the subsequent ones that we plan to add.

Reply:
And what would be an acceptable solution? Copying the macro body to build the nosimd case?